src/lib/Libwin/passwd.c

```diff
@@ -4050,7 +3790,7 @@ setuser(char *user)
 
 	setuser_hdle = LogonUserNoPass(user);
 	if (setuser_hdle == 0) {
-		log_errf(-1, __func__, "failed in LogonUserNoPass for %s", user);
+		log_eventf(PBSEVENT_ERROR, PBS_EVENTCLASS_SERVER, LOG_ERR, __func__, "failed in LogonUserNoPass for %s", user);
```

Comment:
Same question here: Why is log_errf() not good here?

Reply:
Same as above, LogonUserNoPass is PBS API, so it don't set last error.